packages/atlas/atlas.config.yml

```diff
@@ -443,6 +443,8 @@ content:
       name: Thai
     - isoCode: tr
       name: Turkish
+    - isoCode: uk
```

Comment:
```suggestion
    - isoCode: ua
```

Reply:
according to this [list](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) it's `uk`